db/build.gradle

```diff
@@ -44,12 +44,12 @@ android {
 }
 
 dependencies {
-
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.0.4'
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'androidx.core:core-ktx:1.13.1'
     implementation 'org.jetbrains.kotlinx:kotlinx-collections-immutable-jvm:0.3.7'
 
-    api group: 'net.zetetic', name: 'android-database-sqlcipher', version: '4.6.0'
+    api group: 'net.zetetic', name: 'android-database-sqlcipher', version: '4.5.4'
```

Comment:
There's an issue upgrading to the latest version of android-database-sqlcipher?

Reply:
I think I got the 4.6.0 from here: https://github.com/sqlcipher/sqlcipher-android which is the replacement for android-database-sqlcipher